packages/blocks-ui/src/panels/blocks-listing.js

```diff
@@ -9,6 +9,70 @@ import { useEditor } from '../providers/editor'
 import { useBlocks } from '../providers/blocks'
 import { useScope } from '../providers/scope'
 
+// used as a wrapper around like block types
+const BlockSorting = ({ title, children }) => {
+  const [toggle, setToggle] = useState(true)
+  const [height, setHeight] = useState()
+  const ref = useRef(null)
+  useEffect(() => {
+    setHeight(ref.current.offsetHeight)
+  }, [ref])
+  return (
+    <div
+      ref={ref}
+      sx={{
+        boxSizing: 'border-box',
+        width: '100%',
+        border: '1px solid highlight',
+        background: '#F6F6F6', // replace this with 'muted' once it's added to the theme
+        padding: '3',
+        marginBottom: '3',
+        overflow: 'hidden',
+        height: '100%',
+        maxHeight: toggle ? `${height + 52}px` : '52px',
```

Comment:
You don't have to add `px` to the end of the string so you can just use

`maxHeight: toggle ? height + 52 : 52`